rpc/client.ts

```diff
@@ -53,10 +53,11 @@ export class Client<
     } else if (e.id) {
       const pendingCall = this.pendingCalls[e.id];
       if (!pendingCall) {
-        console.log({ e });
+        console.log("!pendingCall for", { e });
         // TODO: pipe error to listeners and message the likely cause,
         //       a duplicate client.
-        throw new Error();
+        // throw new Error();
+        return;
```

Comment:
Is not throwing a good idea? No pending call would signal faulty behavior.

Reply:
will revert